proto/commonfate/control/config/v1alpha1/aws_rds_database.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+package commonfate.control.config.v1alpha1;
+
+import "commonfate/authz/v1alpha1/read_only.proto";
+
+service AWSRDSDatabaseService {
+  rpc CreateAWSRDSDatabase(CreateAWSRDSDatabaseRequest) returns (CreateAWSRDSDatabaseResponse) {}
+  rpc GetAWSRDSDatabase(GetAWSRDSDatabaseRequest) returns (GetAWSRDSDatabaseResponse) {
+    option (commonfate.authz.v1alpha1.read_only) = true;
+  }
+  rpc UpdateAWSRDSDatabase(UpdateAWSRDSDatabaseRequest) returns (UpdateAWSRDSDatabaseResponse) {}
+  rpc DeleteAWSRDSDatabase(DeleteAWSRDSDatabaseRequest) returns (DeleteAWSRDSDatabaseResponse) {}
+}
+
+message CreateAWSRDSDatabaseRequest {
+  // target
+  string name = 1;
+  // role
+  string database_user = 2;
+  string proxy_instance_account = 3;
+  string proxy_instance_region = 4;
+  // this is the actual database name that the proxy connects to
+  string database_name = 5;
+  string ecs_cluster_name = 6;
+  string ecs_service_name = 7;
+  string ecs_container_name = 8;
+}
+
+message AWSRDSDatabase {
+  string id = 1;
+  // target
+  string name = 2;
+  // role
+  string database_user = 3;
```

Comment:
naming nit: it might be clearer if these are just called `target` and `role` 
```suggestion
  // target
  string target = 2;
  // role
  string role = 3;
```